### DISCUSSION TITLE
Does Positron have support for integrating HPC Clusters ?

### DISCUSSION THREAD

**Discussion by srividya76** [posit-dev/positron#5852] [Q&A] [answered]
Does Positron offer support for integration with HPC/Spark clusters and template usage like Posit Workbench now that Positron Pro is part of the Workbench?
There were some issues earlier regarding Positron handling large datasets - is that sorted?
Please let me know.

**Comment by jmcphers** [comment, marked as answer]
Positron does not have specific features for HPC/Spark integration, but Workbench does have some integration for these systems depending on what you're specifically looking for. In general you can expect the Positron Pro preview on Workbench to have similar integration and features as VS Code on Workbench, which has support for running external jobs on HPCs, using Slurm or Kubernetes or Altair for sessions, etc.

Yes, we have recently addressed a number of issues with Positron on large datasets. I'm guessing you're referring to https://github.com/posit-dev/positron/issues/3628 or one of the related issues. We're still making this better, but the most important bits have already landed or will early in the new year.

If you have specific questions about Positron Pro on Workbench after giving it a try, we'd encourage you to get in touch with our support team (support@posit.co).

Thanks for trying Positron!

**Comment by srividya76** [comment]
Thank you !
